jetstream_pt/ray_engine.py

```diff
@@ -230,9 +231,7 @@ def create_pytorch_ray_engine(
     )
   ray.init(ignore_reinit_error=True)
   pod_name = tpu.get_current_pod_name()
-  num_hosts = (
-      num_hosts if is_disaggregated else tpu.get_current_pod_worker_count()
-  )
+  num_hosts = num_hosts if num_hosts > 0 else tpu.get_current_pod_worker_count()
   print(f"pod_name:{pod_name}, number of host: {num_hosts}")
   assert (
```

Comment:
Nit, consider adding more assertion to check the number of hosts * working chips == total devices.

Reply:
Thanks, added.